types/reactstrap/lib/Breadcrumb.d.ts

```diff
@@ -4,6 +4,8 @@ import { CSSModule } from '../index';
 export type BreadcrumbProps<T = {}> = React.HTMLAttributes<HTMLElement> & {
   tag?: string;
   className?: string;
+  listTag?: string;
```

Comment:
Same as the other PR you requested, that's a 6.4 update, you need to increase the version number in the index.d.ts file.